INN/umbrella-currentorg
# Custom ltw templates

## Changes

This pull request makes the following changes:

- Adds a new template `template-page-ltw.php` to use for LTW pages that includes no header or navigation and only a reduced logo
- Updates the `archive-projects.php` archive template to match the look of the new template above
- Creates new stylesheet for said template
- Registers new `current-ltw-template-stylesheet` script and enqueues it where necessary

Shortcode page:
![screencapture-currentorg-test-local-projects-2020-06-25-13_43_03](https://user-images.githubusercontent.com/18353636/85773189-94e7a400-b6eb-11ea-8840-7a6f7c1dc61d.png)

Archive:
![screencapture-currentorg-test-local-that-works-2020-06-25-13_43_22](https://user-images.githubusercontent.com/18353636/85773171-91541d00-b6eb-11ea-9acc-fd4bce62a734.png)

Form page:
![screencapture-currentorg-test-test-form-page-2020-06-25-13_43_12](https://user-images.githubusercontent.com/18353636/85773187-94e7a400-b6eb-11ea-8294-a24013c0a9da.png)

## Why



For #153 

## Testing/Questions

Features that this PR affects:

- Archive template for LTW Projects CPT
- New custom template


Questions that need to be answered before merging:

- [ ] I thought about making `archive-projects.php` just include this new template, but then we'd need to make a ton of conditionals inside of it to distinguish between archive vs. page, and also I wasn't sure if they're ever going to want them to look different. Does it make sense to keep them separate but equal for now?
- [ ] Does the header size look ok @MirandaEcho?

Steps to test this PR:

1. View the LTW archive page and make sure everything still works ok
2. Create a page, add the LTW shortcode to it, and select the new template. View it and make sure everything looks and functions ok
